### PR TITLE
fix: accessability added

### PR DESCRIPTION
**Pull request by jsteenke**

### CODE REVIEW

File: src/app/announcement/announcement-detail/announcement-detail.component.html [hunk 1]
```diff
@@ -121,7 +125,7 @@
                 tooltipPosition="top"
                 tooltipEvent="hover"
               ></p-calendar>
-              <label class="ocx-required-label" for="announcement_detail_form_startDate">
+              <label id="date-from-label" class="ocx-required-label" for="announcement_detail_form_startDate">
```
Comment (HenryT-CG) [inline src/app/announcement/announcement-detail/announcement-detail.component.html:128]: not a valid id in this dialog ... check other id declarations

File: src/app/announcement/announcement-detail/announcement-detail.component.html [hunk 2]
```diff
@@ -63,12 +64,14 @@
...
           <p-dropdown
             id="announcement_detail_form_workspace_name"
+            aria-label="'ANNOUNCEMENT.WORKSPACE' | translate"
             class="input-field"
             styleClass="w-full"
             [options]="allWorkspaces"
             [autoDisplayFirst]="true"
             [appendTo]="'body'"
             formControlName="workspaceName"
+            optionLabel="'ANNOUNCEMENT.WORKSPACE' | translate"
```
Comment (HenryT-CG) [inline src/app/announcement/announcement-detail/announcement-detail.component.html:74]: pls. list aria-label and optionLabel behind each ohter

File: src/app/announcement/announcement-detail/announcement-detail.component.html [hunk 3]
```diff
@@ -150,14 +154,17 @@
...
+                {{ 'ANNOUNCEMENT.END_DATE' | translate }}</label
+              >
             </span>
           </div>
         </div>
         <div class="col-6 sm:col-2 py-0 sm:pl-4">
           <div class="w-1 flex flex-column row-gap-2 mr-2">
             <p-inputSwitch
               id="announcement_detail_form_status"
+              inputId="announcement_detail_form_status"
```
Comment (HenryT-CG) [inline src/app/announcement/announcement-detail/announcement-detail.component.html:167]: id AND inputId ?
In this case, I think, id is obsolet

File: src/app/announcement/announcement-detail/announcement-detail.component.html [hunk 4]
```diff
@@ -177,7 +184,7 @@
             formGroup.controls['endDate'].errors?.hasOwnProperty('invalidDateRange')
           "
           class="col-12 py-0 ml-1 sm:pl-3 control-error"
-          for="announcement_detail_form_start_date"
+          for="announcement_detail_form_startDate"
```
Comment (HenryT-CG) [inline src/app/announcement/announcement-detail/announcement-detail.component.html:187]: do NOT use camel case syntax in id fields

File: src/app/announcement/announcement-search/announcement-search.component.html [hunk 5]
```diff
@@ -174,6 +178,7 @@
     <div class="flex flex-wrap column-gap-2 row-gap-1 justify-content-end">
       <button
         pButton
+        (click)="displayDeleteDialog = false"
```
Comment (HenryT-CG) [inline src/app/announcement/announcement-search/announcement-search.component.html:181]: move this line after iconPos

File: src/assets/i18n/de.json [hunk 6]
```diff
@@ -33,6 +33,7 @@
       }
     },
     "EDIT": {
+      "LABEL": "Neuigkeit bearbeiten",
```
Comment (HenryT-CG) [inline src/assets/i18n/de.json:36]: Duplication?
Why you added a new item here, with same content as the following line?

File: src/app/announcement/announcement-detail/announcement-detail.component.html (hunk 3 above)
Comment (jsteenke) [reply src/app/announcement/announcement-detail/announcement-detail.component.html:167]: id is the id for p-inputSwitch. inputId forwards an id to <input> element inside <p-inputSwitch>. Either delete id or rename inputId ?

**Review by HenryT-CG — APPROVED** [review]

**Review by HenryT-CG — APPROVED** [review]

**Pull request merged**